kolibri/core/content/migrations/0016_auto_20190124_1639.py

```diff
@@ -4,13 +4,23 @@
 
 from django.db import migrations
 
-from kolibri.core.content.utils.annotation import calculate_next_order
+
+def calculate_next_order(channel, model):
```

Comment:
Instead of duplicating the logic here, maybe we can explicitly pass the model in `kolibri.core.content.utils.annotation.calculate_next_order` as you have done here.